.github/workflows/ci.yaml

```diff
@@ -33,3 +33,66 @@ jobs:
 
       - name: Run tox
         run: tox -e lint,py,coverage-report,typing  # run tox using Python in path
+
+  build:
+    runs-on: ubuntu-latest
+    needs: [test]
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Define the Docker tag
+        id: vars
+        run: echo ::set-output name=tag::$(echo ${GITHUB_REF} | sed -E 's,refs/(heads|tags)/,,' | sed -E 's,/,-,g')
+
+      - name: Print the tag
+        id: print
+        run: echo ${{steps.vars.outputs.tag}}
+
+      - name: Log into Docker Hub
+        run: echo ${{ secrets.DOCKER_TOKEN }} | docker login --username ${{ secrets.DOCKER_USERNAME }} --password-stdin
+
+      - name: Pull previous images
+        run: |
+          docker pull lsstsqre/neophile:deps-${{steps.vars.outputs.tag}} || true
+          docker pull lsstsqre/neophile:${{steps.vars.outputs.tag}} || true
+
+      - name: Build the dependencies Docker image
+        run: |
+          docker build --target dependencies-image \
+            --cache-from=lsstsqre/neophile:deps-${{steps.vars.outputs.tag}} \
+            --tag lsstsqre/neophile:deps-${{steps.vars.outputs.tag}} .
+
+      - name: Build the runtime Docker image
+        run: |
+          docker build --target runtime-image \
+            --cache-from=lsstsqre/neophile:${{steps.vars.outputs.tag}} \
+            --tag lsstsqre/neophile:${{steps.vars.outputs.tag}} .
+
+      - name: Push Docker images
+        run: |
+          docker push lsstsqre/neophile:deps-${{steps.vars.outputs.tag}}
+          docker push lsstsqre/neophile:${{steps.vars.outputs.tag}}
+
+  docs:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2.0.1
+        with:
+          python-version: 3.8
+
+      - name: Install tox and LTD Conveyor
+        run: pip install tox ltd-conveyor
+
+      - name: Run tox
+        run: tox -e docs
+
+      - name: Upload to LSST the Docs
+        env:
+          LTD_USERNAME: travis
```

Comment:
Use `${{ secrets.LTD_USERNAME }}` instead. The `travis` was from an early transition project, but `LTD_USERNAME`/`LTD_PASSWORD` are available to all projects in the organization.